### DISCUSSION TITLE
Accessing past performance notifications via SP-API

### DISCUSSION THREAD

**Discussion by spencerkent** [amzn/selling-partner-api-models#3465] [General] [answered]
The [performance notifications page in Seller Central](https://sellercentral.amazon.com/performance/notifications) allows one to see **past** performance notifications—my question is: how can this information be accessed through the SP-API? I realize that once I've set up a subscription to certain notifications via the [Notifications API](https://developer-docs.amazon.com/sp-api/docs/notifications-api-v1-reference), this will allow me to see notifications **in the future** as they arrive, but the question is whether I can see past notifications too, or if there is another way to access such data. I'm thinking that most message queueing services like SQS don't by default retain old messages after a certain amount of time—is that the case for the queues underlying the notifications API?

**Comment by wanglei0214** [comment, marked as answer]
Hi spencerkent,

Thanks for asking. The past performance can not be accessed even if you have set up a subscription of Notification through SP-API. The underlying mechanism between the performance notifications page in Seller Central compared to Notification API are not the same. And as for SQS service information, please check AWS FAQs.

Thanks,
Lei
